person.rb

```diff
@@ -0,0 +1,24 @@
+class Person
+  attr_reader :id, :name, :age
```

Comment:
- [ ] Let's encapsulate _name_ and _age_ variables _with **attr_accessor**_ as we intend to both and read and modify their values.

Reply:
It's Done.